src/microbe_stage/gui/PatchDetailsPanel.cs

```diff
@@ -154,6 +154,12 @@ public class PatchDetailsPanel : PanelContainer
 
     public bool IsPatchMoveValid { get; set; }
 
+    public bool MoveToPatchButtonVisible
```

Comment:
Can be an export variable.

Reply:
I'm having some trouble with the Export attribute.

Problem - With the export attribute present, the property seems to try to set itself before nodes have entered the tree.

What I have tried:
 - Null check return if null (The button and HSplit are always visible the first time regardless of the editor setting but updates properly when changed during runtime)
 - Null check with GetNode to fill the null fields before setting visibility (Null reference error NodePath is null)
 
 I'm still new to Godot so I might just be missing something simple.

Reply:
That's known. Our common solution seems to be:

1. Add a corresponding field
2. Set the field when property is set; set visibility if not null
3. Explicitly set visibility in _Ready()

2 and 3 logic can be merged into an update function